gdx/src/com/badlogic/gdx/utils/Select.java

```diff
@@ -35,7 +35,7 @@ public class Select {
 	private QuickSelect quickSelect;
 
 	/** Provided for convenience */
-	public static Select instance () {
+	public synchronized static Select instance () {
```

Comment:
What's the point of `synchronized` here?

Reply:
singleton pattern. What happens when multiple threads try to access instance() ?